test/tester.go

```diff
@@ -0,0 +1,206 @@
+package test
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/NebulousLabs/skynet-accounts/api"
+	"github.com/NebulousLabs/skynet-accounts/database"
+	"github.com/NebulousLabs/skynet-accounts/email"
+	"github.com/NebulousLabs/skynet-accounts/jwt"
+	"github.com/NebulousLabs/skynet-accounts/metafetcher"
+	"github.com/sirupsen/logrus"
+	"gitlab.com/NebulousLabs/errors"
+)
+
+var (
+	testPortalAddr = "http://127.0.0.1"
+	testPortalPort = "6000"
+	pathToJWKSFile = "../../jwt/fixtures/jwks.json"
+)
+
+type (
+	// AccountsTester is a simple testing kit for accounts. It starts a testing
+	// instance of the service and provides simplified ways to call the handlers.
+	AccountsTester struct {
+		Ctx    context.Context
```

Comment:
Unexporting a field and locking it behind an exported getter is a standard practice, yes. The reason for that is that often we want to limit or validate the way users interact with otherwise sensitive internal fields. Having in mind that this is a simple tester, I don't think the overhead of adding extra methods here is worth it. I would rather keep it as simple as possible, unless there are validations that I keep doing. If that's the case I'd put those in a getter at that point.